js/components/geolocation.js

```diff
@@ -3,7 +3,8 @@ Fliplet.FormBuilder.field('geolocation', {
   category: 'Advanced',
   props: {
     value: {
-      type: String
+      type: Array,
```

Comment:
Why is it that we need to change the variable type for `value` and can't just keep the `accuracy`?

Reply:
I tried to create an `accuracy` property, but I needed to update the value in `form.js` file. I developed an emit method, but somehow it didn't work with `_input` event. If this solution doesn't please you, I can try finding another solution for `accuracy` prop.

Reply:
👍🏻